roles/rundeck/tasks/admin-credentials.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Ensure that rundeck_admin_credentials has been set
+  ansible.builtin.assert:
+    that: (rundeck_admin_credentials != "")
+    fail_msg: please set rundeck_admin_credentials
+    success_msg: rundeck_admin_credentials has been set
+
+- name: Parse rundeck_admin_credentials
+  ansible.builtin.set_fact:
+    rundeck_admin_credential_parts: '{{ rundeck_admin_credentials | split(":") }}'
+
+- name: Generate credential facts
```

Comment:
What is this for? I don't see any of it being used

Reply:
Faulty implementation I did this morning, it's marked `outdated` by GH and if it isn't gone I'll erase it in a bit